pipelines/docker-build-oci-ta/README.md

```diff
@@ -95,7 +95,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |name|description|default value|already set by|
 |---|---|---|---|
 |AUTH_TOKEN_COVERITY_IMAGE| Name of secret which contains the authentication token for pulling the Coverity image.| auth-token-coverity-image| |
-|COV_LICENSE| Name of secret which contains the Coverity license| cov-license| |
+|COV_LICENSE| Name of secret which contains the Coverity license. Only required for public Konflux instances.| cov-license| |
```

Comment:
We may mention `It automatically get coverity license from internal URL for internal Konflux, no need to set for internal Konflux` or something like this  in descriptions.
Others look good to me.